<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterFace</class>
 <widget class="QWidget" name="InterFace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="Time">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QLabel" name="UserName">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Status">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>status</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>260</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color : 'blue'</string>
   </property>
   <property name="text">
    <string>See More</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AccptRequestBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>130</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Accept Request</string>
   </property>
  </widget>
  <widget class="QPushButton" name="CurrentTripBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Current Trip</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SetStatusBtn">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>70</y>
     <width>121</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Set Status</string>
   </property>
  </widget>
  <widget class="QLabel" name="Address">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Address</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="LogOutBtn">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>0</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Log Out</string>
   </property>
  </widget>
  <widget class="QCommandLinkButton" name="RefreshBtn">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Passenger1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Passenger2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Passenger3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="Passenger4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>391</width>
     <height>27</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="Name1">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="source1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>290</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QLabel" name="cost1">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="distance1">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="Name2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="source2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>320</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QLabel" name="cost2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="distance2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>320</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="distance3">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="distance4">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>380</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>distance</string>
   </property>
  </widget>
  <widget class="QLabel" name="cost4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>380</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="cost3">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>cost</string>
   </property>
  </widget>
  <widget class="QLabel" name="source4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QLabel" name="source3">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>350</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Source</string>
   </property>
  </widget>
  <widget class="QLabel" name="Name4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="Name3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SeeMore">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>350</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>641</width>
     <height>411</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>b1.jpg</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>Time</zorder>
  <zorder>UserName</zorder>
  <zorder>Status</zorder>
  <zorder>label_1</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>label_8</zorder>
  <zorder>AccptRequestBtn</zorder>
  <zorder>CurrentTripBtn</zorder>
  <zorder>SetStatusBtn</zorder>
  <zorder>Address</zorder>
  <zorder>LogOutBtn</zorder>
  <zorder>RefreshBtn</zorder>
  <zorder>Name1</zorder>
  <zorder>source1</zorder>
  <zorder>cost1</zorder>
  <zorder>distance1</zorder>
  <zorder>Name2</zorder>
  <zorder>source2</zorder>
  <zorder>cost2</zorder>
  <zorder>distance2</zorder>
  <zorder>distance3</zorder>
  <zorder>distance4</zorder>
  <zorder>cost4</zorder>
  <zorder>cost3</zorder>
  <zorder>source4</zorder>
  <zorder>source3</zorder>
  <zorder>Name4</zorder>
  <zorder>Name3</zorder>
  <zorder>SeeMore</zorder>
  <zorder>Passenger1</zorder>
  <zorder>Passenger2</zorder>
  <zorder>Passenger3</zorder>
  <zorder>Passenger4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
